### PR TITLE
Update locked oak item description

### LINKED ISSUE #295: The description of Oak Items is misleading
When the player does not have enough Pokémon for an Oak item, the message is 'Capture X Pokémon to unlock.' It should be 'Capture X unique Pokémon to unlock.' as that is the real requirement.

### PR DESCRIPTION
**Pull request by RedSparr0w**
Closes #295 
Screenshot from 2019-05-30 11-54-24

### CODE REVIEW

**Comment by Ishadijcks** [comment]
Ooh I like that. It is not what I meant but it's good!

File: src/index.html [hunk 1]
```diff
@@ -610,7 +610,7 @@ <h1>Pokéclicker</h1>
                         </div>
 
                         <div style="min-height:80px; vertical-align: center">
-                            Capture <span data-bind="text: OakItemRunner.inspectedItem().unlockReq"></span> Pokémon to
+                            Capture <span data-bind="text: OakItemRunner.inspectedItem().unlockReq - player.caughtPokemonList.length"></span> new Pokémon to
```
Comment (Ishadijcks) [inline src/index.html]:
```suggestion
                            Capture <span data-bind="text: OakItemRunner.inspectedItem().unlockReq - player.caughtPokemonList.length"></span> more unique Pokémon to
```

**Comment by Ishadijcks** [comment]
Does this work properly with Knockout? As in does the number update whenever I'm looking at an item and I catch another Pokemon?

**Comment by RedSparr0w** [comment]
Yup, sure does! :+1:

**Comment by Ishadijcks** [comment]
Great!

**Pull request merged**